### PR TITLE
Fixing error learning rate for this sample

### PR DESCRIPTION
**Pull request by Hierom**
@MorvanZhou 
Thanks for giving this kind tutorial for everyone who is learning pytorch.
By the way, there are one syntax warning should be fixed in your code for the future version pytorch (>0.5).

> Use tensor.item() to convert a 0-dim tensor to a Python number

Another one problem is about the learning rate for this example, lr = 0.5 was too big that will not changing loss in this example, so I revise it as 0.4.

### CODE REVIEW

**Comment by Hierom** [comment]
I saw that you are fixed these error in the contents folder, so let's revise it in the same way.
Thanks
